_data/projects/Basic-Car-Maintenance.yml

```
@@ -0,0 +1,9 @@
+name: Basic Car Maintenance
+desc: A brief description of the project
+site: https://github.com/mikaelacaron/Basic-Car-Maintenance
+tags:
+  - swift
+  - firebase
+upforgrabs:
+  name: good first issue
+  link: https://github.com/mikaelacaron/Basic-Car-Maintenance/issues?q=is:open+is:issue+label:%22good+first+issue%22
```

:art: a simpler URL to this list of issues

```suggestion
  link: https://github.com/mikaelacaron/Basic-Car-Maintenance/labels/good%20first%20issue
```